README.md

```diff
@@ -35,12 +35,13 @@ GET /:path
 
 ### Environment variables
 
-| Name        | Description                                                                             |
-|-------------|-----------------------------------------------------------------------------------------|
-| PORT        | The port to listen to requests on                                                       |
-| TOKEN       | GitHub Personal Access Token used for authorization when pulling the content repository. (Only needed if the content repo is private) |
-| CONTENT_URL | The repository to get content from                                                      |
-| CONTENT_DIR | Directory to serve contents from. Setting this disables the automatic fetching using git and makes the `TOKEN` and `CONTENT_URL` unused. |
+| Name         | Description                                                                                                                              |
+|--------------|------------------------------------------------------------------------------------------------------------------------------------------|
+| PORT         | The port to listen to requests on                                                                                                        |
+| TOKEN        | GitHub Personal Access Token used for authorization when pulling the content repository. (Only needed if the content repo is private)    |
+| CONTENT_URL  | The repository to get content from                                                                                                       |
+| CONTENT_DIR  | Directory to serve contents from. Setting this disables the automatic fetching using git and makes the `TOKEN` and `CONTENT_URL` unused. |
+| DARKMODE_URL | URL to the darkmode system, or `true` or `false` to use that value instead of sending an http request.                                   |
```

Comment:
We will have to add this to the `docker-compose` file in `bawang-content`, and perhaps also an instance of the darkmode system? Some way of quickly testing darkmode for the website would be nice, but that is a problem for later.

see [bawang-content#514](https://github.com/datasektionen/bawang-content/pull/514)

Reply:
It's probably easier to put `true` or `false` in the url instead of using the actual darkmode system

Reply:
lol, I didn't read the readme apparently. Missed that it was possible to do that :p

Reply:
Well sure, but it should always just return a boolean, the benefit is so minimal I don't think it's worth it. Although the effort is also pretty small, but still ^^